k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_cluster.tpl

```diff
@@ -33,6 +33,8 @@ cluster:
     - name: utils
     - name: hare
     - name: rgw
+      services:
+        - rgw_s3
```

Comment:
Any reason why the sample file has not been updated?

https://github.com/Seagate/cortx-prvsnr/blob/2fc7486a652db805619525fab5c26bed98d5a22b/conf/solution/cluster.yaml.sample#L36-L40

Reply:
@keithpine , We have made the sample files changes now for rgw_s3. Please check once.

Reply:
Looks good, thanks!